imagebuildah/build.go

```diff
@@ -136,11 +139,14 @@ type Executor struct {
 	reportWriter                   io.Writer
 }
 
-func makeSystemContext(signaturePolicyPath string) *types.SystemContext {
+func makeSystemContext(signaturePolicyPath string, tlsVerify bool) *types.SystemContext {
 	sc := &types.SystemContext{}
 	if signaturePolicyPath != "" {
 		sc.SignaturePolicyPath = signaturePolicyPath
 	}
+	if tlsVerify {
```

Comment:
Why is this check necessary?  This says it will only modify the systemcontext if tleVerify is true?  Why not always set it below.

Reply:
You're right, removing it.  I should not have had that there.